pydoctor/templates/apidocs.css

```diff
@@ -182,13 +182,13 @@ code > .code {
 }
 
 
-div.function {
+#childList > div {
```

Comment:
I'm not a super fan of "div" in selectors. It's not exactly self-documenting. `#childList` at least I sort of know what that means.  Not a blocker though, this is all CSS, YOLO